frontend/src/metabase/static-viz/components/XYChart/types.ts

```diff
@@ -15,13 +16,28 @@ export type YAxisPosition = "left" | "right";
 
 export type VisualizationType = "line" | "area" | "bar" | "waterfall";
 
-export type Series = {
-  name: string;
-  color: string;
+interface BaseSeries {
+  name: string | null;
   data: SeriesData;
   type: VisualizationType;
   yAxisPosition: YAxisPosition;
-};
+}
+
+export interface SeriesWithOneOrLessDimensions extends BaseSeries {
```

Comment:
These new 2 types are explained in this comment. https://github.com/metabase/metabase/pull/26211/files#r1026373144